xclim/indicators/land/_streamflow.py

```diff
@@ -59,74 +61,80 @@ def cfcheck(self, **das):
 
 
 base_flow_index = Streamflow(
+    title="Base flow index",
     identifier="base_flow_index",
     units="",
     long_name="Base flow index",
-    description="Minimum 7-day average flow divided by the mean flow.",
+    description="Minimum of the 7-day moving average flow divided by the mean flow.",
+    asbtract="Minimum of the 7-day moving average flow divided by the mean flow.",
     compute=base_flow_index,
 )
 
 freq_analysis = FA(
+    title="Return period flow amount",
     identifier="freq_analysis",
     var_name="q{window}{mode:r}{indexer}",
-    long_name="N-year return period {mode} {indexer} {window}-day flow",
-    description="Streamflow frequency analysis for the {mode} {indexer} {window}-day flow "
-    "estimated using the {dist} distribution.",
+    long_name="N-year return period flow amount",
+    description="Streamflow frequency analysis for the {mode} {indexer} {window}-day flow estimated using the {dist} "
+    "distribution.",
+    abstract="Streamflow frequency analysis on the basis of a given mode and distribution.",
     units="m^3 s-1",
-    title="Flow values for given return periods.",
     compute=declare_units(da=None)(frequency_analysis),
 )
 
 rb_flashiness_index = Streamflow(
+    title="Richards-Baker flashiness index",
     identifier="rb_flashiness_index",
     units="",
     var_name="rbi",
     long_name="Richards-Baker flashiness index",
-    description="{freq} R-B Index, an index measuring the flashiness of flow.",
+    description="{freq} de R-B Index, an index measuring the flashiness of flow.",
+    abstract="Measurement of flow oscillations relative to average flow, "
+    "quantifying the frequency and speed of flow changes.",
     compute=rb_flashiness_index,
 )
 
 stats = Stats(
+    title="Statistic of the daily flow for a given period.",
     identifier="stats",
     var_name="q{indexer}{op:r}",
-    long_name="{freq} {op} of {indexer} daily flow ",
+    long_name="Daily flow statistics",
     description="{freq} {op} of {indexer} daily flow",
```

Comment:
```suggestion
    description="{freq} {op} of daily flow ({indexer})",
```
To sync with the suggested french.